CONTRIBUTING.md

```diff
@@ -49,10 +49,24 @@ this will get rid of the errors.
 
     The errors should now be gone.
 
+### C) Setting up the JDT Language Server in JDT Language Server
```

Comment:
Maybe `Setting up the JDT Language Server in other clients` ?

If you feel "other" is a bit dismissive we could maybe use something like "lightweight editors", or "vim/emacs clients" for now.

Reply:
Or perhaps "Setting up the JDT Language server in nvim or emacs"

or perhaps two sections, one for each?

Reply:
Another question, @rgrunber and @schrieveslaach , JDTLS is configured automatically in lsp-mode, but it just works for gradle or maven projects. It doesn't understand PDE projects, by default. Is that the case also in nvim, @schrieveslaach ? If so, should this section clarify that these instructions just are needed for getting PDE support?